__tests__/__packages__/cli-test-utils/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI test utils package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Updated peer dependencies to `^8.0.0`, dropping support for next-tagged versions [#2287](https://github.com/zowe/zowe-cli/pull/2287)
```

Comment:
Please end sentence w/ a period.
Can we say "... for versions tagged `next`"? If accurate, I think it'd be more clear.